hack/build-go.sh

```diff
@@ -16,6 +16,10 @@
 
 # This script is a vestigial redirection.  Please do not add "real" logic.
 
+# This script runs `make all` command.
+# The command compiles all Makefile configs.
+# Usage: `hack/build-go.sh`.
```

Comment:
I would call out that this uses the WHAT env var and provide an additional example